ui/app/components/app/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -35,6 +35,7 @@ export default class TransactionListItemDetails extends PureComponent {
     rpcPrefs: PropTypes.object,
     senderAddress: PropTypes.string.isRequired,
     tryReverseResolveAddress: PropTypes.func.isRequired,
+    addressBook: PropTypes.array,
```

Comment:
As noted below, can we move the logic for receiving `senderNickname` into the container? Then we can change this to `senderNickname` and make it required.

Reply:
For the tests we can pass in whatever silly string

Reply:
@whymarrh nickname logic is now in the container, `recipientNickname` would also in this case need to be a required prop with this change. I think this is what you meant, let me know!